src/parser.js

```diff
@@ -427,7 +427,21 @@ var Parser = Object.extend({
     },
 
     parseInclude: function() {
-        return this.parseTemplateRef('include', nodes.Include);
```

Comment:
After this PR, `parseTemplateRef` is only used by `parseExtends`. I think we may as well just remove `parseTemplateRef` entirely in this PR.